packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -4,204 +4,549 @@
+              items: [
+                for (var alignment in alignmentEnumEntries)
+                  DropdownMenuItem(
+                    value: alignment,
+                    child: Container(
+                      child: Row(
+                        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                        crossAxisAlignment: CrossAxisAlignment.center,
+                        children: <Widget>[
+                          Text(
+                            describeEnum(alignment),
+                            style: TextStyle(
+                              color: color,
+                            ),
+                          ),
+                          Image.asset(
+                            (axis == direction)
+                                ? getMainAxisAssetImageUrl(alignment)
+                                : getCrossAxisAssetImageUrl(alignment),
+                            height: 24,
+                            fit: BoxFit.contain,
+                          ),
+                        ],
+                      ),
+                    ),
+                  )
+              ],
+              onChanged: (Object newSelection) {
+                setState(() {
+                  if (axis == direction) {
+                    mainAxisAlignment = newSelection;
+                  } else {
+                    crossAxisAlignment = newSelection;
+                  }
+                });
+              },
+            ),
+          )
+        ],
       ),
     );
   }
 
   @override
   Widget build(BuildContext context) {
-    final flexType = properties.type.toString();
     final theme = Theme.of(context);
     return Container(
       padding: const EdgeInsets.only(top: 32.0),
       child: Column(
         mainAxisAlignment: MainAxisAlignment.start,
         children: [
           Container(
-            margin: const EdgeInsets.only(bottom: 36.0),
+            margin: const EdgeInsets.only(bottom: 24.0),
             child: Text(
               'Story of the flex layout of your $flexType widget',
               style: theme.textTheme.headline,
               textAlign: TextAlign.center,
             ),
           ),
-          Expanded(
+          Align(
+            alignment: Alignment.centerRight,
+            child: _buildAxisAlignmentDropdown(Axis.horizontal),
+          ),
+          Flexible(
             child: LayoutBuilder(builder: (context, constraints) {
-              final children = _visualizeChildren(context);
-              final flexDirectionWrapper = Flex(
-                direction: properties.direction,
-                children: children,
-              );
-              final childrenVisualizerWidget = Column(
-                crossAxisAlignment: CrossAxisAlignment.start,
-                children: [
-                  Container(
-                    child: Text(
-                      properties.type.toString(),
+              final maxHeight = constraints.maxHeight * 0.95;
```

Comment:
why .95? add an explanation and a todo to remove.

Reply:
nice catch! Thanks Jacob.
This was a misuse because I was lazy to set a margin at the time, dropping this one.